baseapp/config.go

```diff
@@ -14,10 +14,16 @@
 
 package baseapp
 
+type TLSConfig struct {
+	CertificateFile string `yaml:"cert_file" json:"certFile"`
+	PrivateKeyFile  string `yaml:"private_key_file" json:"privateKeyFile"`
```

Comment:
The standard library calls these "cert file" and "key file". Should we match that terminology here? The abbreviated `cert_file` matched with the expanded `private_key_file` is bothering me.

Reply:
I'll change them to match the standard library for consistency